static/app/views/performance/database/modulePageProviders.tsx

```diff
@@ -9,13 +9,14 @@ import {RoutingContextProvider} from 'sentry/views/starfish/utils/routingContext
 interface Props {
   children: React.ReactNode;
   title: string;
+  baseURL?: string;
 }
 
-export function ModulePageProviders({title, children}: Props) {
+export function ModulePageProviders({title, children, baseURL}: Props) {
   const organization = useOrganization();
 
   return (
-    <RoutingContextProvider value={{baseURL: '/performance/database'}}>
+    <RoutingContextProvider value={{baseURL: baseURL || '/performance/database'}}>
```

Comment:
I don't think we should default to database here, but there are so many places that this is used (even places where it might not make sense to use database such as interactions module). We should probably go over and fix the baseUrl everywhere and not put a default in a seperate PR.

Reply:
good idea, something def worth refactoring. cc @gggritso

Reply:
Agreed! We also _could_ remove this altogether!

1. Move all current "Starfish" initiatives under `/performance/` routing, since `/starfish/` never ships anyway. Delete "Database", and move "Interactions"
2. Remove `RoutingContext` completely, also use `/performance/` as the base, fix whatever code this breaks

Reply:
Yeah that makes sense to me @gggritso!